<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsInfoWindow</class>
 <widget class="QMainWindow" name="GraphicsInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics info</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_mode">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_mode">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Графики</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_expressions"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_n3">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="shortcut">
          <string>3</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_delLast">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;-</string>
         </property>
         <property name="shortcut">
          <string>Backspace</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_mull">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>*</string>
         </property>
         <property name="shortcut">
          <string>*</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_n2">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="shortcut">
          <string>2</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_close">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>)</string>
         </property>
         <property name="shortcut">
          <string>)</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_n0">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="shortcut">
          <string>0</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_pluss">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="shortcut">
          <string>+</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_n9">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="shortcut">
          <string>9</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_n7">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="shortcut">
          <string>7</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_n5">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="shortcut">
          <string>5</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_n4">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_n6">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_n1">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="shortcut">
          <string>1</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_n8">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="shortcut">
          <string>8</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_znak">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButton_minus">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="shortcut">
          <string>-</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_open">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>(</string>
         </property>
         <property name="shortcut">
          <string>(</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_devide">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="shortcut">
          <string>/</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButton_add">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_dot">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>,</string>
         </property>
         <property name="shortcut">
          <string>,</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_pow">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>^</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_functions">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>f(x)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_x">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>x</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
         <property name="autoRepeatDelay">
          <number>50</number>
         </property>
         <property name="autoRepeatInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Опции</string>
    </property>
    <addaction name="action_open_painter"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_open_painter">
   <property name="text">
    <string>Открыть рисовальщик</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
